Cmdline/Action/KSP.cs

```diff
@@ -550,36 +564,48 @@ private int FakeNewKSPInstall (FakeOptions options)
             {
                 User.RaiseError("Couldn't find a valid KSP version for your input.\n" +
                 	"Make sure to enter the at least the version major and minor values in the form Maj.Min - e.g. 1.5");
+                User.RaiseMessage("--Error--");
                 return Exit.BADOPT;
             }
             catch (CancelledActionKraken)
             {
                 User.RaiseError("Selection cancelled! Please call 'ckan ksp fake' again.");
+                User.RaiseMessage("--Error--");
                 return Exit.ERROR;
             }
 
-
             User.RaiseMessage(String.Format("Creating new fake KSP install {0} at {1} with version {2}", installName, path, version.ToString()));
+            log.Debug("Faking instance...");
 
             try
             {
                 // Pass all arguments to CKAN.KSPManager.FakeInstance() and create a new one.
-                Manager.FakeInstance(installName, path, version, dlcVersion);
-                if (setToDefault)
+                Manager.FakeInstance(installName, path, version, dlcVersions);
+                if (setDefault)
```

Comment:
We are missing curly braces here. Only the `RaiseMessage` call is controlled by the `if`, the `SetAutoStart` will happen unconditionally.

Reply:
A bit of Python here and there can't hurt.

... actually, it does in this case.